code/__HELPERS/game.dm

```diff
@@ -276,12 +276,15 @@
 	else
 		return 0
 
-
-/proc/try_move_adjacent(atom/movable/AM)
+/proc/try_move_adjacent(atom/movable/AM, trydir)
 	var/turf/T = get_turf(AM)
+	if(trydir)
+		if(AM.Move(get_step(T, trydir)))
+			return TRUE
 	for(var/direction in GLOB.cardinals)

```

Comment:
-= trydir maybe?

Reply:
actually nevermind that'll fuck the glob

Reply:
Yes, Here can be used &